src/arrayMethodUnshift.js

```diff
@@ -5,7 +5,14 @@
  */
 function applyCustomUnshift() {
   [].__proto__.unshift2 = function(...elements) {
-    // write code here
+    this.reverse();
+
+    for (let i = elements.length; i > 0;) {
+      this[this.length] = elements[i - 1];
+      i = i - 1;
```

Comment:
add this to `for` condition, no need to add this separately,
you can write this `let i = elements.length` like `let i = elements.length - 1` (and fix the rest respectively)